<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Agency FB</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>电话：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_male">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>132</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_female">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>132</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_account">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phone">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
